governance/polls/Adjust Spark D3M Parameters - September 4, 2023.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Adjust Spark Protocol D3M Parameters - September 4, 2023
+summary: Signal your support or opposition to adjusting the Spark Protocol DAI Direct Deposit Module (D3M) parameters.
+discussion_link: https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-09-04T16:00:00
+end_date: 2023-09-07T16:00:00
+---
+# Poll: Adjust Spark Protocol D3M Parameters - September 4, 2023
+
+The Reserve Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Stability Facilitators. The community can vote in this poll to express support or opposition to using the Spark Protocol Direct Deposit DAI Module (D3M) with the listed parameters. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, September 4 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to making the following parameter changes to the Spark Protocol DAI Direct Deposit Module (D3M):
+* Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 200 million DAI from 200 million DAI to **400 million DAI**.
```

Comment:
Minor nitpick, but we would tend to include the technical variable name as well

so "Maximum Debt Ceiling (`line`)" - helps if people are reviewing exec code at any point to be familiar with var names.

Again, not blocking